x-pack/plugins/apm/public/components/app/settings/agent_configurations/agent_configuration_create_edit/service_page/service_page.tsx

```diff
@@ -111,20 +97,15 @@ export function ServicePage({ newConfig, setNewConfig, onClickNext }: Props) {
           'xpack.apm.agentConfig.servicePage.service.fieldLabel',
           { defaultMessage: 'Service name' }
         )}
-        isLoading={serviceNamesStatus === FETCH_STATUS.LOADING}
-        options={serviceNameOptions}
+        esField={SERVICE_NAME}
         value={newConfig.service.name}
```

Comment:
What about calling it `fieldName` instead of `esField`?

```suggestion
        fieldName={SERVICE_NAME}
        value={newConfig.service.name}
```

Reply:
updated https://github.com/elastic/kibana/pull/122648/commits/43e7793ffd1ae6bf45f1eeac7b8ecee9b47215e4